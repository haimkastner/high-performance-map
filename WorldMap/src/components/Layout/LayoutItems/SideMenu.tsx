import React from 'react';

export class SideMenu extends React.Component {

    render() {
        return (<h1>SideMenu</h1>);
    }
}
