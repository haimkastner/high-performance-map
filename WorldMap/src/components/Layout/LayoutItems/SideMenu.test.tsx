import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { SideMenu} from './SideMenu';

describe('SideMenu Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SideMenu />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<SideMenu />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // tslint:disable-next-line: no-empty
  describe('SideMenu Component Logic', () => {});
});
