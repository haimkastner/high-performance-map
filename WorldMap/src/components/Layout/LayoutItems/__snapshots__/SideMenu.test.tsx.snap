// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu Component matches the snapshot 1`] = `
<h1>
  SideMenu
</h1>
`;
